model/registry/vulnerability.yaml

```diff
@@ -0,0 +1,120 @@
+        examples: ['CVE']
+      - id: id
+        type: string
+        stability: experimental
+        brief: >
+          The number portion of a vulnerability entry. It includes a unique identification number for the vulnerability.
+          For example, [Common Vulnerabilities and Exposure CVE ID](https://www.cve.org/ResourcesSupport/FAQs).
+        examples: ['CVE-2021-44228']
+      - id: reference
+        type: string
+        stability: experimental
+        brief: >
+          A resource that provides additional information, context, and mitigations for the identified vulnerability.
+        examples: ['https://www.cve.org/CVERecord?id=CVE-2021-44228']
+      - id: report_id
+        type: string
+        stability: experimental
+        brief: >
+          The report or scan identification number.
+        examples: ['20191018.0001']
+      - id: scanner.vendor
+        type: string
+        stability: experimental
+        brief: >
+          The name of the scanner vendor that identified the vulnerability.
+        examples: ['Tenable']
+      - id: score.base
+        type: double
+        stability: experimental
+        brief: >
+          The base score of the vulnerability.
+        note: >
+          Scores can range from 0.0 to 10.0, with 10.0 being the most severe. Base scores cover an assessment
+          for exploitability metrics (attack vector, complexity, privileges, and user interaction),
+          impact metrics (confidentiality, integrity, and availability), and scope. For example, [CVSS](https://www.first.org/cvss/specification-document).
+
+          The National Vulnerability Database (NVD) provides qualitative severity rankings of "Low", "Medium",
+          and "High" for CVSS v2.0 base score ranges in addition to the severity ratings for CVSS v3.0 as they are
+          defined in the CVSS v3.0 specification.
+        examples: [7.5]
+      - id: score.environmental
+        type: double
+        stability: experimental
+        brief: >
+          The environmental score of the vulnerability.
+        note: >
+          Scores can range from 0.0 to 10.0, with 10.0 being the most severe. Environmental scores cover an assessment
+          for any modified base metrics, confidentiality, integrity, and availability requirements. For example, [CVSS](https://www.first.org/cvss/specification-document).
+        examples: [7.5]
+      - id: score.temporal
+        type: double
+        stability: experimental
+        brief: >
+          The temporal score of the vulnerability.
+        note: >
+          Scores can range from 0.0 to 10.0, with 10.0 being the most severe. Temporal scores cover an assessment for
+          code maturity, remediation level, and confidence. For example, [CVSS](https://www.first.org/cvss/specification-document).
+        examples: [7.5]
+      - id: score.version
```

Comment:
Do we always expect to have only one scoring system represented in the vulnerability?
For example, what if a vendor has its own scoring system and also adds the cvssv2 and cvssv3 info? 
Should we allow to store all this information?

Reply:
hey @leykin-valeriy I have provided 2 options in the comment above on how to include multiple standards into namespace, you can give you opinion there, which option do you like more (or maybe have another idea on how accomplish multiple standards)